shared/chat/conversation/input-area/filepicker-popup/index.native.js

```diff
@@ -16,16 +17,60 @@ const promptContainerStyle = {
 }
 
 class FilePickerPopup extends React.Component<Props> {
-  onImage = () => {
-    this.props.onSelect('photo')
-  }
-
-  onVideo = () => {
-    this.props.onSelect('video')
-  }
-
   render() {
-    const items = [...[{onClick: this.onImage, title: 'Photo'}], ...[{onClick: this.onVideo, title: 'Video'}]]
+    const items = isIOS
+      ? [
+          ...[
```

Comment:
i know this is based on the old code but this... isn't necessary at all, can be
```
items = isIOS ? [
 {
              onClick: () => {
                this.props.onSelect('photo', 'camera')
              },
              title: 'Take Photo',
            },
{
              onClick: () => {
                this.props.onSelect('mixed', 'library')
              },
              title: 'Photo or Video from Library',
            },
]
```

or even better the whole block can be

```
const items = [
    {
        onClick: () => { this.props.onSelect('photo', 'camera') },
        title: 'Take Photo',
    },
    ...(isIOS 
    ? [{
        onClick: () => { this.props.onSelect('mixed', 'library') },
        title: 'Photo or Video from Library',
      }] 
    : [{
        onClick: () => { this.props.onSelect('video', 'camera') },
        title: 'Take Video',
    },
    {
        onClick: () => { this.props.onSelect('photo', 'library') },
        title: 'Photo from Library',
    },
    {
        onClick: () => { this.props.onSelect('video', 'library') },
        title: 'Video from Library',
    },
    ])
]


```

Reply:
Dropped the `...`, but kept it separate, I think it is nice to be able to read it clearly for each platform.